src/exodus/tx.cpp

```diff
@@ -81,6 +81,22 @@ std::string exodus::strTransactionType(uint16_t txType)
     }
 }
 
+static int EnsureEnableSigma(uint32_t propertyId)
+{
+    if (!IsPropertyIdValid(propertyId)) {
+        PrintToLog("%s(): rejected: property %d does not exist\n", __func__, propertyId);
+        return PKT_ERROR_TOKENS - 24;
```

Comment:
We should not hard-coded error category here due to it may incompatible with the category in the caller.